lib/ansible/modules/monitoring/zabbix/zabbix_user.py

```diff
@@ -0,0 +1,583 @@
+            lang=dict(type='str', default='en_GB', choices=['en_GB', 'en_US', 'zh_CN', 'cs_CZ', 'fr_FR',
+                                                            'he_IL', 'it_IT', 'ko_KR', 'ja_JP', 'nb_NO',
+                                                            'pl_PL', 'pt_BR', 'pt_PT', 'ru_RU', 'sk_SK',
+                                                            'tr_TR', 'uk_UA']),
+            theme=dict(type='str', default='default', choices=['default', 'blue-theme', 'dark-theme']),
+            autologin=dict(type='bool', default=False),
+            autologout=dict(type='str', default='0'),
+            refresh=dict(type='str', default='30'),
+            rows_per_page=dict(type='str', default='50'),
+            after_login_url=dict(type='str', default=''),
+            user_medias=dict(type='list', required=True,
+                             elements='dict',
+                             options=dict(
+                                 mediatype=dict(type='str', default='Email'),
+                                 sendto=dict(type='str', required=True),
+                                 period=dict(type='str', default='1-7,00:00-24:00'),
+                                 severity=dict(type='dict',
+                                               options=dict(
+                                                   not_classified=dict(type='bool', default=True),
+                                                   infomation=dict(type='bool', default=True),
+                                                   warning=dict(type='bool', default=True),
+                                                   average=dict(type='bool', default=True),
+                                                   high=dict(type='bool', default=True),
+                                                   disaster=dict(type='bool', default=True)),
+                                               default=dict(
+                                                   not_classified=True,
+                                                   infomation=True,
+                                                   warning=True,
+                                                   average=True,
+                                                   high=True,
+                                                   disaster=True
+                                               )),
+                                 active=dict(type='bool', default=True)
+                             )),
+            type=dict(type='str', default='Zabbix user', choices=['Zabbix user', 'Zabbix admin', 'Zabbix super admin']),
+            state=dict(type='str', default="present", choices=['present', 'absent']),
+            timeout=dict(type='int', default=10)
+        ),
+        supports_check_mode=True
+    )
+
+    if not HAS_ZABBIX_API:
+        module.fail_json(msg="Missing required zabbix-api module (check docs or install with: pip install zabbix-api)")
```

Comment:
There has been an effort to unify module dependency errors under common helper. Please see #58002 for more information.
```suggestion
        module.fail_json(msg=missing_required_lib('zabbix-api', url='https://pypi.org/project/zabbix-api/'), exception=ZBX_IMP_ERR)
```

Reply:
Thank you for your advice!
I took that advice and added missing_required_lib helper.